contracts/test/VBep20MockDelegate.sol

```diff
@@ -183,6 +183,12 @@ contract VBep20MockDelegate is VToken, VBep20Interface {
         return token.balanceOf(address(this));
     }
 
+    function setStableInterestRateModel(StableRateModel newStableInterestRateModel) public returns (uint256) {
+        stableRateModel = newStableInterestRateModel;
+        return uint(Error.NO_ERROR);
+        //return _setStableInterestRateModelFresh(newStableInterestRateModel);
```

Comment:
remove this line 🤔

Reply:
Fixed in this [commit](https://github.com/VenusProtocol/venus-protocol/pull/210/commits/46f0cf168db69632d3f6a108f589965f5544c579)